src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/RecommendationService.kt

```diff
@@ -220,18 +221,25 @@ internal class RecommendationService(
           createdBy = readableUserName
         )
 
+      if (existingRecommendationEntity.data.managerRecallDecision?.isSentToDelius == true) {
+        log.info("About to send domain event for ${updatedRecommendation.crn} on manager recall decision made for recommendationId $recommendationId")
+        try {
+          sendManagerRecallDecisionMadeEvent(
+            crn = existingRecommendationEntity.data.crn,
+            contactOutcome = existingRecommendationEntity.data.managerRecallDecision?.selected?.value.toString(),
+            staffcode = getValueAndHandleWrappedException(userId?.let { communityApiClient.getStaffDetails(it) })?.staffCode
+          )
+        } catch (ex: Exception) {
+          log.info("Failed to send domain event for ${updatedRecommendation.crn} on manager recall decision for recommendationId $recommendationId")
+          existingRecommendationEntity.data.managerRecallDecision?.isSentToDelius = false
+          log.info("Reverting isSentToDelius flag to false")
+          updateAndSaveRecommendation(existingRecommendationEntity, userId, readableUserName)
```

Comment:
You're saving the reommendation twice which doesn't seem right? It does it on line 241